### PR TITLE
Add next and prev buttons to docs

### PR DESCRIPTION
**Pull request by virgilt**
Closes https://github.com/taskcluster/taskcluster/issues/1376.

### CODE REVIEW

**Comment by virgilt** [comment]
@djmitche I tried to add a changelog file under changelog/ as specified but am getting an error. Says has no body at snippets.

**Comment by virgilt** [comment]
@djmitche Nevermind, I was able to find another changelog commit.

**Comment by virgilt** [comment]
@helfi92 @djmitche Not sure what to do for codecov/project failure. I looked at the failing files and I did not edit any of them.

**Comment by virgilt** [comment]
@djmitche @helfi92 Removed the changelog, so this can be reviewed when ready.

File: ui/src/views/Documentation/index.jsx [hunk 1]
```diff
@@ -75,10 +75,9 @@ export default class Documentation extends Component {
   }
 
   findChildFromRootNode(node) {
-    const currentPath = window.location.pathname.replace(
-      `${DOCS_PATH_PREFIX}/`,
-      ''
-    );
+    const currentPath = window.location.pathname
+      .replace(/\/+$/, '')
```
Comment (helfi92) [inline ui/src/views/Documentation/index.jsx]: What's the reason for adding `+`? I wonder if `.replace(/\/$/, '')` is all we need.

Comment (virgilt) [reply ui/src/views/Documentation/index.jsx]: @helfi92 You are right. No need for the +. I removed it.

**Pull request merged**